datastore/additional_data/management/commands/rewrite_additional_data.py

```diff
@@ -1,8 +1,16 @@
+from __future__ import annotations
```

Comment:
I'm quite strongly against `__future__` as it blurs the python version compatibility and can add code into the codebase which introduces features in an inconsistent way. If we want to commit to using annotations we can a) decide that b) upgrade python version.

Reply:
That's a point! I'd accidentally left it in from earlier experiments anyway, so removed.